### DISCUSSION TITLE
Vue Meta InnerHTML

### DISCUSSION THREAD

**Discussion by mirabledictu** [nuxt/nuxt#17540] [Questions] [answered]
Hi, is nuxt3 still using `vue-meta`? If yes, how can we set `style` with innerHTML?

I tried this but it didnt work

```ts
export default defineNuxtConfig({
    meta: {
        style: [
            {  vmid: 'someId', content: '.foo { color: red }', type: 'text/css' } // or hid instead of vmid
        ],
        __dangerouslyDisableSanitizersByTagID: {
            'someId': ['innerHTML']
        }
    }
})
```

The output is

```html
<style vmid="someId" content=".foo { color: red }" type="text/css"></style>
```

I expect the content to go inside the tags

**Comment by danielroe** [comment, marked as answer]
Believe it or not, it's as simple at the moment as:
```js
import { defineNuxtConfig } from 'nuxt3';

export default defineNuxtConfig({
  meta: {
    style: [{ children: ':root { color: red }', type: 'text/css' }],
  },
})
```

However, **please note that this is an area that is likely to change**, which is also why it's not fully documented.

**Reply by mirabledictu** [reply]
Holy crap, thanks. Better than before!

**Comment by wobsoriano** [comment]
Hi @danielroe , I tried to do this inside a module and it doesn't seem to work?

```ts
export default defineNuxtModule({
  meta: {
    name: "something",
    configKey: "something"
  },
  defaults: {},
  async setup(options, nuxt) {
    nuxt.options.meta.style.push({
      type: 'text/css',
      children: ':root { color: red }'
    })
  }
});
```

I checked the `head` element and I don't see it there but when I manually add it to nuxt config, it works flawlessly.

**Reply by danielroe** [reply]
It's been moved to `app.head`.
